Build fails on Windows

As I don't have an access to *nix machine for about a month (long story), attempting to build on Windows:
```
Running `rustc --crate-name taskchampion_cli --edition=2018 cli\src\lib.rs 
...
error[E0433]: failed to resolve: could not find `unix` in `os`
  --> cli\src\lib.rs:34:14
   |
34 | use std::os::unix::ffi::OsStringExt;
   |              ^^^^ could not find `unix` in `os`

error[E0599]: no method named `into_vec` found for struct `OsString` in the current scope
  --> cli\src\lib.rs:66:42
   |
66 |         .map(|oss| String::from_utf8(oss.into_vec()))
   |                                          ^^^^^^^^ method not found in `OsString`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `taskchampion-cli`
```
Is there an OS-agnostic replacement for [std::os::unix::ffi](https://doc.rust-lang.org/std/os/unix/ffi/index.html)?.. How important is building on a non-unix? 
@djmitche , do you aim for TC to be operational on Windows/MacOS?

--- a/cli/src/lib.rs
+++ b/cli/src/lib.rs
@@ -31,8 +31,7 @@ For the public TaskChampion Rust API, see the `taskchampion` crate.
 
 */
 
-use std::os::unix::ffi::OsStringExt;
-use std::string::FromUtf8Error;
+use std::ffi::OsString;
 
 // NOTE: it's important that this 'mod' comes first so that the macros can be used in other modules
 mod macros;
@@ -63,8 +62,8 @@ pub fn main() -> Result<(), Error> {
     // parse the command line into a vector of &str, failing if
     // there are invalid utf-8 sequences.
     let argv: Vec<String> = std::env::args_os()
-        .map(|oss| String::from_utf8(oss.into_vec()))
-        .collect::<Result<_, FromUtf8Error>>()
+        .map(|oss| oss.into_string())
+        .collect::<Result<_, OsString>>()
         .map_err(|_| Error::for_arguments("arguments must be valid utf-8"))?;
     let argv: Vec<&str> = argv.iter().map(|s| s.as_ref()).collect();
 

